paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -427,6 +429,11 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         _paymentSheetResult.value = PaymentSheetResult.Canceled
     }
 
+    fun initializeResourceRepository() {
```

Comment:
this could be avoided if `ResourceRepository` does the logic in its native `init` block